docs/tutorials/java-troubleshooting.rst

```diff
@@ -34,7 +38,11 @@ You should receive a message (by default in Slack) with the troubleshooting opti
   :align: center
 
 
+| From here you can debug the memory in your Java application with jmap or diagnose problems with thread activity via jstack.
+| Check out :ref:`Java Troubleshooting Actions` for more information on what each of the debugging choices do.
+
```

Comment:
Maybe include a picture below and write "For example, here is jmap output showing a summary of the application's memory usage"